### PR TITLE
[OPL] GPSR rules based on new Command Generator

### PR DESCRIPTION
**Pull request by MrKeith99**
### Major Changes:
- Delete Category 1~3
- Delete previous `Speech (ask)`
- Add `Speech (tell)`
- Add `Vision (Description)`

### Proposal:
- Not use `greetNameInRm` and `greetClothDscInRm` commands (lower difficulty).
- If `DEM Category` is requested, *0.5* deduction is applied as `Weight`.

### Request:
- `$POSE/GESTURE`: list of possible pose/gesture of a person
- `$OBJ_COMP`: list of object characteristics
- `PERS_INFO`: name list and `$POSE/GESTURE` list
- `QUESTION_LIST` : list of possible questions from guest
- `$TELL_LIST` : list of information given from the robot

### CODE REVIEW

File: rules/OPL/gpsr_ja.md [hunk 1]
```diff
@@ -122,43 +123,47 @@ Step 1を完了している場合，リスタート後に直前のタスクを
...
+### カテゴリごとの重み (Weights per Category)
 
 | カテゴリ | $Weight$ |
 | -------- | ------ |
-| 0 | $0.7$ |
-| 1 | $1$ |
-| 2 | $2$ |
-| 3 | $3$ |
+| DEM | 0.5 |
+| 通常 | 1 |
```
Comment (akinobu1998) [inline rules/OPL/gpsr_ja.md:160]: 重み一の場合，合計点500点獲得可能でしょうか．
重みかベーススコアを調整する必要があると思います．

Comment (MrKeith99) [reply rules/OPL/gpsr_ja.md:160]: その通りです．確認します．

Comment (MrKeith99) [reply rules/OPL/gpsr_ja.md:160]: https://github.com/RoboCupAtHomeJP/AtHome2024/pull/23/commits/eb49104447bac7e87053a68de10ee281e37f9d37

Comment (MrKeith99) [reply rules/OPL/gpsr_ja.md:160]: `通常カテゴリ`の場合，500点満点が取れるようにしました．

**Pull request merged**